plugins/timelapse.py

```diff
@@ -267,8 +267,7 @@ def start(self):
 
     def acquire(self, dlg):
         main_data = self.main_app.main_data
-        str_ctrl = main_data.tab.value.streambar_controller
```

Comment:
You cannot remove these lines, as they don't do the same thing as `.pause()`. These lines ensure that no stream is live.